client/ayon_houdini/plugins/publish/collect_render_products.py

```diff
@@ -168,6 +169,15 @@ def get_aov_identifier(self, render_product):
             str: The AOV identifier
 
         """
+        # Allow explicit naming through custom attribute on the render product
+        aov_identifier = render_product.GetPrim().GetAttribute(
+            "ayon:aov_identifier").Get(time=Usd.TimeCode.EarliestTime())
+        if aov_identifier:
```

Comment:
After some discussion, let's allow setting identifiers with empty strings which will override the beauty path.
```suggestion
        aov_identifier = render_product.GetPrim().GetAttribute("ayon:aov_identifier")
        if aov_identifier:
            aov_identifier = aov_identifier.Get(time=Usd.TimeCode.EarliestTime())
```